google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -159,6 +160,13 @@ public static TransactionOption optimisticLock() {
     return OPTIMISTIC_LOCK_OPTION;
   }
 
+  /**
+   * Specifying this instructs the transaction to request {@link IsolationLevel} from the backend.
+   */
+  public static IsolationLevelOption isolationLevelOption(IsolationLevel isolationLevel) {
```

Comment:
nit: the return type here should be `TransactionOption` instead of `IsolationLevelOption`. The latter is not a public class or interface, but this method is.